src/test/java/org/mdbenefits/app/journeys/MdBenefitsFlowJourneyTest.java

```diff
@@ -15,941 +15,1006 @@
-        assertThat(testPage.hasErrorText(message("error.missing-general")));
-
-        // should redirect to MyMDTHINK
-        testPage.clickElementById("applicationInfo-" + ApplicantObjective.COLLEGE_STUDENT_IN_APP.name());
-        testPage.clickContinue();
-        assertThat(testPage.getTitle()).isEqualTo(message("redirect.mdthink.title"));
-
-        // should not redirect
-        testPage.navigateToFlowScreen("mdBenefitsFlow/selectApplication");
-        testPage.clickElementById("none__checkbox");
-        testPage.clickContinue();
-        assertThat(testPage.getTitle()).isEqualTo(message("help-needed.title"));
-    }
-
-    @Test
-    void redirectToMyMDTHINKOnUnsupportedCounty() {
-        testPage.navigateToFlowScreen("mdBenefitsFlow/county");
-        // should redirect to MyMDTHINK
-        testPage.selectFromDropdown("county", "Frederick County");
-        testPage.clickContinue();
-        assertThat(testPage.getTitle()).isEqualTo(message("redirect.mdthink.title"));
-
-        // should not redirect
-        testPage.navigateToFlowScreen("mdBenefitsFlow/county");
-        testPage.selectFromDropdown("county", "Baltimore County");
-        testPage.clickContinue();
-        assertThat(testPage.getTitle()).isEqualTo(message("select-app.title"));
-    }
-
-    @Test
-    void selectHelpNeededAndChooseProgramFlow() {
-        // select help needed
-        testPage.navigateToFlowScreen("mdBenefitsFlow/selectHelpNeeded");
-        testPage.clickContinue();
-        assertThat(testPage.getTitle()).isEqualTo("Select help");
-        assertThat(testPage.hasErrorText(message("error.missing-general")));
+        protected static final String RANGE_ERROR_MESSAGE = "Make sure to provide a value between 1 and 100.";
+
+        @Test
+        void redirectToMyMDTHINKOnUnsupportedApplicationType() {
+                testPage.navigateToFlowScreen("mdBenefitsFlow/selectApplication");
+                testPage.clickContinue();
+                assertThat(testPage.getTitle()).isEqualTo("Select application");
+                assertThat(testPage.hasErrorText(message("error.missing-general")));
+
+                // should redirect to MyMDTHINK
+                testPage.clickElementById("applicationInfo-" + ApplicantObjective.COLLEGE_STUDENT_IN_APP.name());
+                testPage.clickContinue();
+                assertThat(testPage.getTitle()).isEqualTo(message("redirect.mdthink.title"));
+
+                // should not redirect
+                testPage.navigateToFlowScreen("mdBenefitsFlow/selectApplication");
+                testPage.clickElementById("none__checkbox");
+                testPage.clickContinue();
+                assertThat(testPage.getTitle()).isEqualTo(message("help-needed.title"));
+        }
+
+        @Test
+        void redirectToMyMDTHINKOnUnsupportedCounty() {
+                testPage.navigateToFlowScreen("mdBenefitsFlow/county");
+                // should redirect to MyMDTHINK
```

Comment:
I'm guessing you meant to only have 4 space indents here?  Can you revert the indent change?  Thanks!

Reply:
I haven't fully read this file yet as the diff is noisy.  I'll read it on Monday.

Reply:
Thank you!